Add page for rsh

YuTingLan opened the pull request:
新增飯店、景點、住宿分頁（6個一頁）
![截圖 2023-05-09 下午9 42 03](https://github.com/TourMind/tourmind/assets/127638342/272756b0-3705-44a6-9721-f54b0676b281)
![截圖 2023-05-09 下午9 42 55](https://github.com/TourMind/tourmind/assets/127638342/91aa7497-887c-4b6e-9584-8ef1f3c51d7a)
![截圖 2023-05-09 下午9 43 09](https://github.com/TourMind/tourmind/assets/127638342/9e178e32-5f38-4a35-bca9-6a0abed1abd3)

YuTingLan committed:
add navbar

YuTingLan committed:
add navbar

YuTingLan committed:
add navbar

YuTingLan committed:
add navbar

YuTingLan committed:
used rubocop

YuTingLan committed:
mail send is changed from google to mailgun

YuTingLan committed:
mail send is changed from google to mailgun

YuTingLan committed:
modify mailgun api key

YuTingLan committed:
WIP

YuTingLan committed:
WIP

YuTingLan committed:
WIP

YuTingLan committed:
Can't switch branches, only commit

YuTingLan committed:
add dashboard_users page

YuTingLan committed:
add dashboard_users page

YuTingLan committed:
add dashboard_users page

YuTingLan committed:
add dashboard_users page

YuTingLan committed:
dashboard_users Done

YuTingLan committed:
dashboard_users Done

YuTingLan committed:
modify path

YuTingLan committed:
modify path

YuTingLan committed:
modify path

YuTingLan committed:
used rubocop

YuTingLan committed:
Remove unnecessary paths and add a background link

YuTingLan committed:
Remove unnecessary paths and add a background link

YuTingLan committed:
first rebase

YuTingLan committed:
first rebase

YuTingLan committed:
first rebase

CCTWeng committed:
Favorite list (#51)

* Changing column type and updaing input form

cuisine types and atmostphere are string of array that can take multiples input

fix card

WIP front-end design

finish with restaurant type

* update Show UI

* partial rendering for Show page

* get location with geocoder and activejob

* add google map

* partial rendering for Show page

rubucop

redriect after create

* rubocup

* get location with geocoder and add google map

* fix image height

* add validation

* rubocop

* fix image ratio

* fix conflict

* fix credential

* add basic favorite list function

* rubocop

* check user login

* update tailwindcss

* WIP favorite list index page

* toggle favorite button

remove console log

rubocop

* WIP fix reload page

* add section filter

* minor fix

* finish basic section filter

* WIP subtoggle

* WIP index.html for favorite list

* make favorite button an object

* fix redirect to top of the page

* check

* WIP favorite list index page

* index page for favorite list

index page for favorite list

* fix layout

* fix conflict

* add association with hotel and site

* add link for testing function

* fix typo

* add authorization

* fix credentails

* Update database.yml

CCTWeng committed:
Favorite list (#51)

* Changing column type and updaing input form

cuisine types and atmostphere are string of array that can take multiples input

fix card

WIP front-end design

finish with restaurant type

* update Show UI

* partial rendering for Show page

* get location with geocoder and activejob

* add google map

* partial rendering for Show page

rubucop

redriect after create

* rubocup

* get location with geocoder and add google map

* fix image height

* add validation

* rubocop

* fix image ratio

* fix conflict

* fix credential

* add basic favorite list function

* rubocop

* check user login

* update tailwindcss

* WIP favorite list index page

* toggle favorite button

remove console log

rubocop

* WIP fix reload page

* add section filter

* minor fix

* finish basic section filter

* WIP subtoggle

* WIP index.html for favorite list

* make favorite button an object

* fix redirect to top of the page

* check

* WIP favorite list index page

* index page for favorite list

index page for favorite list

* fix layout

* fix conflict

* add association with hotel and site

* add link for testing function

* fix typo

* add authorization

* fix credentails

* Update database.yml

CCTWeng committed:
Favorite list (#51)

* Changing column type and updaing input form

cuisine types and atmostphere are string of array that can take multiples input

fix card

WIP front-end design

finish with restaurant type

* update Show UI

* partial rendering for Show page

* get location with geocoder and activejob

* add google map

* partial rendering for Show page

rubucop

redriect after create

* rubocup

* get location with geocoder and add google map

* fix image height

* add validation

* rubocop

* fix image ratio

* fix conflict

* fix credential

* add basic favorite list function

* rubocop

* check user login

* update tailwindcss

* WIP favorite list index page

* toggle favorite button

remove console log

rubocop

* WIP fix reload page

* add section filter

* minor fix

* finish basic section filter

* WIP subtoggle

* WIP index.html for favorite list

* make favorite button an object

* fix redirect to top of the page

* check

* WIP favorite list index page

* index page for favorite list

index page for favorite list

* fix layout

* fix conflict

* add association with hotel and site

* add link for testing function

* fix typo

* add authorization

* fix credentails

* Update database.yml

CCTWeng committed:
Favorite list (#51)

* Changing column type and updaing input form

cuisine types and atmostphere are string of array that can take multiples input

fix card

WIP front-end design

finish with restaurant type

* update Show UI

* partial rendering for Show page

* get location with geocoder and activejob

* add google map

* partial rendering for Show page

rubucop

redriect after create

* rubocup

* get location with geocoder and add google map

* fix image height

* add validation

* rubocop

* fix image ratio

* fix conflict

* fix credential

* add basic favorite list function

* rubocop

* check user login

* update tailwindcss

* WIP favorite list index page

* toggle favorite button

remove console log

rubocop

* WIP fix reload page

* add section filter

* minor fix

* finish basic section filter

* WIP subtoggle

* WIP index.html for favorite list

* make favorite button an object

* fix redirect to top of the page

* check

* WIP favorite list index page

* index page for favorite list

index page for favorite list

* fix layout

* fix conflict

* add association with hotel and site

* add link for testing function

* fix typo

* add authorization

* fix credentails

* Update database.yml

YuTingLan committed:
delete duplicate program of JS index

YuTingLan committed:
add background judgment

YuTingLan committed:
fix rebase conflicts

YuTingLan committed:
fix rebase conflicts

YuTingLan committed:
fix rebase conflicts

YuTingLan committed:
fixed rebase issues

YuTingLan committed:
remove unnecessary photos

YuTingLan committed:
Merge branch 'dev' into dashboard_rebase_dev

YuTingLan committed:
fixed migration timing conflict.

YuTingLan committed:
Merge branch 'dev' of dashboard_rebase_dev

YuTingLan committed:
resolve rebase conflicts

YuTingLan committed:
added role column to user

YuTingLan committed:
added faker data and fixed order show issue

YuTingLan committed:
E

YuTingLan committed:
add excon (~> 0.71)

YuTingLan committed:
delete uploaders img

YuTingLan committed:
delete uploaders img

YuTingLan committed:
Merge branch 'dev' into dashboard_rebase_dev

YuTingLan committed:
modify Uploader missing problem

YuTingLan committed:
modify Uploader missing problem

YuTingLan committed:
Merge branch 'dev' into dashboard_rebase_dev

YuTingLan committed:
resolve merge conflicts

YuTingLan committed:
Remove the "p" for the test

YuTingLan committed:
added hotel, site, restaurant page using pagy

YuTingLan committed:
added hotel, site, restaurant page using pagy

YuTingLan committed:
added hotel, site, restaurant page using pagy

chencac committed:
AWS

chencac committed:
flx

chencac committed:
show_guide

chencac committed:
show_guide

Hsubetty committed:
index keyword input and newbtn update

Hsubetty committed:
index keyword input and newbtn update

chencac committed:
show_guide

Hsubetty committed:
update keyword input and remove new button

Hsubetty committed:
update keyword input and remove new button

Hsubetty committed:
update keyword input and remove new button

Hsubetty committed:
update inquiry button color

Hsubetty committed:
update inquiry button color

Hsubetty commented on app/views/sites/index.html.erb:
```diff
@@ -125,9 +125,7 @@
-      <div>
-        <%= paginate @sites%>
-      </div>
+      <div class="flex justify-center mt-4"><%= raw pagy_nav(@pagy) %></div>
```
想到因index會有篩選功能～可以改為如果沒有飯店資訊就不顯示pagy
   <%if @hotels.count > 0 %>
      <div class="flex justify-center mt-4"><%= raw pagy_nav(@pagy) %></div>
   <% end %>

YuTingLan commented:
餐廳/景點/住宿為超過6個才做分頁
正修組員提供意見，小於6個的時候不要顯示分頁

YuTingLan committed:
Condition not met, no page break style displayed.

YuTingLan committed:
Condition not met, no page break style displayed.

chencac committed:
home_plan

chencac committed:
home_plan

chencac committed:
home_plan_OK

chencac committed:
home_plan_OK

YuTingLan committed:
Plans limit (#82)

* add plans limit feature

* add Membership Expiry Date judge

* total_comments

* rating_bug

* plan_rating

* plan_rating

* comment_flx

* rating_test

* commit

* add seed

* update show view

* update index view and seed format

* delete useless images

* fixed password reset issue

* hotels layout

* hotels index layout finished

* yarn add fontawesome and fiexed controller

* mutiple uploder images

* hotel show layout WIP

* modify hotel list style align to sites page and restaurant page

* hotels add filter function

* update the index bg color and removed show 評等

* update the filter bug and add google map

* added delete picture function

* add favorite_list logic

* fix hotel delete bug and add image validate

* update the error message and view RWD

* update image_uploader and error message and update validates

* remove image_uploader minimagic setting and show view

* reconfirm 404 and run rubocop

* fix devise format

* Sites  (#52)

* sites layout

* sites layout WIP

* sites layout finished

* fixed model and view sites index layout finished

* yarn add fontawesome

* sites add filter function

* modify sites model, add images column, rename pet_friendly column

* site show layout-update fake image logic

* rename pet_friendly to sites

* update the filter coloumn 科博館

* update index and remove show 評等

* add goodle map

* rename the site.controller.js to filter.controller.js and run rubocop

* add delete picture function

* add favorite_list logic

* fix site delete bug and add duration validates

* add image validate

* update the text color

* update the error message and view RWD

* update image_uploader and error message

* remove image_uploader minimagic setting and show view

* show all the upload images

* fix: 修正衝突錯誤

* format devise.zh.yml

* fix show sites

* fix show sites

* fix show sites

* add favoorite icon

* index and show add dynamic star and rating by comments

* add parking checkbox in form

* update star and rating bug

* fix image cache

* delete useless images

* friendly

* plan_friendly

* add editorial judgment

---------

Co-authored-by: chen <zx656084@gmail.com>
Co-authored-by: Betty Hsu <997666aa@gmail.com>
Co-authored-by: Hsu Betty <123808430+Hsubetty@users.noreply.github.com>

YuTingLan committed:
Plans limit (#82)

* add plans limit feature

* add Membership Expiry Date judge

* total_comments

* rating_bug

* plan_rating

* plan_rating

* comment_flx

* rating_test

* commit

* add seed

* update show view

* update index view and seed format

* delete useless images

* fixed password reset issue

* hotels layout

* hotels index layout finished

* yarn add fontawesome and fiexed controller

* mutiple uploder images

* hotel show layout WIP

* modify hotel list style align to sites page and restaurant page

* hotels add filter function

* update the index bg color and removed show 評等

* update the filter bug and add google map

* added delete picture function

* add favorite_list logic

* fix hotel delete bug and add image validate

* update the error message and view RWD

* update image_uploader and error message and update validates

* remove image_uploader minimagic setting and show view

* reconfirm 404 and run rubocop

* fix devise format

* Sites  (#52)

* sites layout

* sites layout WIP

* sites layout finished

* fixed model and view sites index layout finished

* yarn add fontawesome

* sites add filter function

* modify sites model, add images column, rename pet_friendly column

* site show layout-update fake image logic

* rename pet_friendly to sites

* update the filter coloumn 科博館

* update index and remove show 評等

* add goodle map

* rename the site.controller.js to filter.controller.js and run rubocop

* add delete picture function

* add favorite_list logic

* fix site delete bug and add duration validates

* add image validate

* update the text color

* update the error message and view RWD

* update image_uploader and error message

* remove image_uploader minimagic setting and show view

* show all the upload images

* fix: 修正衝突錯誤

* format devise.zh.yml

* fix show sites

* fix show sites

* fix show sites

* add favoorite icon

* index and show add dynamic star and rating by comments

* add parking checkbox in form

* update star and rating bug

* fix image cache

* delete useless images

* friendly

* plan_friendly

* add editorial judgment

---------

Co-authored-by: chen <zx656084@gmail.com>
Co-authored-by: Betty Hsu <997666aa@gmail.com>
Co-authored-by: Hsu Betty <123808430+Hsubetty@users.noreply.github.com>

YuTingLan committed:
Plans limit (#82)

* add plans limit feature

* add Membership Expiry Date judge

* total_comments

* rating_bug

* plan_rating

* plan_rating

* comment_flx

* rating_test

* commit

* add seed

* update show view

* update index view and seed format

* delete useless images

* fixed password reset issue

* hotels layout

* hotels index layout finished

* yarn add fontawesome and fiexed controller

* mutiple uploder images

* hotel show layout WIP

* modify hotel list style align to sites page and restaurant page

* hotels add filter function

* update the index bg color and removed show 評等

* update the filter bug and add google map

* added delete picture function

* add favorite_list logic

* fix hotel delete bug and add image validate

* update the error message and view RWD

* update image_uploader and error message and update validates

* remove image_uploader minimagic setting and show view

* reconfirm 404 and run rubocop

* fix devise format

* Sites  (#52)

* sites layout

* sites layout WIP

* sites layout finished

* fixed model and view sites index layout finished

* yarn add fontawesome

* sites add filter function

* modify sites model, add images column, rename pet_friendly column

* site show layout-update fake image logic

* rename pet_friendly to sites

* update the filter coloumn 科博館

* update index and remove show 評等

* add goodle map

* rename the site.controller.js to filter.controller.js and run rubocop

* add delete picture function

* add favorite_list logic

* fix site delete bug and add duration validates

* add image validate

* update the text color

* update the error message and view RWD

* update image_uploader and error message

* remove image_uploader minimagic setting and show view

* show all the upload images

* fix: 修正衝突錯誤

* format devise.zh.yml

* fix show sites

* fix show sites

* fix show sites

* add favoorite icon

* index and show add dynamic star and rating by comments

* add parking checkbox in form

* update star and rating bug

* fix image cache

* delete useless images

* friendly

* plan_friendly

* add editorial judgment

---------

Co-authored-by: chen <zx656084@gmail.com>
Co-authored-by: Betty Hsu <997666aa@gmail.com>
Co-authored-by: Hsu Betty <123808430+Hsubetty@users.noreply.github.com>

YuTingLan committed:
Plans limit (#82)

* add plans limit feature

* add Membership Expiry Date judge

* total_comments

* rating_bug

* plan_rating

* plan_rating

* comment_flx

* rating_test

* commit

* add seed

* update show view

* update index view and seed format

* delete useless images

* fixed password reset issue

* hotels layout

* hotels index layout finished

* yarn add fontawesome and fiexed controller

* mutiple uploder images

* hotel show layout WIP

* modify hotel list style align to sites page and restaurant page

* hotels add filter function

* update the index bg color and removed show 評等

* update the filter bug and add google map

* added delete picture function

* add favorite_list logic

* fix hotel delete bug and add image validate

* update the error message and view RWD

* update image_uploader and error message and update validates

* remove image_uploader minimagic setting and show view

* reconfirm 404 and run rubocop

* fix devise format

* Sites  (#52)

* sites layout

* sites layout WIP

* sites layout finished

* fixed model and view sites index layout finished

* yarn add fontawesome

* sites add filter function

* modify sites model, add images column, rename pet_friendly column

* site show layout-update fake image logic

* rename pet_friendly to sites

* update the filter coloumn 科博館

* update index and remove show 評等

* add goodle map

* rename the site.controller.js to filter.controller.js and run rubocop

* add delete picture function

* add favorite_list logic

* fix site delete bug and add duration validates

* add image validate

* update the text color

* update the error message and view RWD

* update image_uploader and error message

* remove image_uploader minimagic setting and show view

* show all the upload images

* fix: 修正衝突錯誤

* format devise.zh.yml

* fix show sites

* fix show sites

* fix show sites

* add favoorite icon

* index and show add dynamic star and rating by comments

* add parking checkbox in form

* update star and rating bug

* fix image cache

* delete useless images

* friendly

* plan_friendly

* add editorial judgment

---------

Co-authored-by: chen <zx656084@gmail.com>
Co-authored-by: Betty Hsu <997666aa@gmail.com>
Co-authored-by: Hsu Betty <123808430+Hsubetty@users.noreply.github.com>

CCTWeng committed:
標籤功能 (#87)

* show tag when using filter

* add tag link

* add tag remove function

* add close icon

add close icon

* add tags to hotels & sites & rubocup

* fix typo

* fix conflict

YuTingLan committed:
Dashboard rebase dev (#55)

* WIP

* add dashboard_users page

* dashboard_users Done

* modify path

* used rubocop

* Remove unnecessary paths and add a background link

* mail send is changed from google to mailgun

* modify mailgun api key

* add navbar

* used rubocop

* Can't switch branches, only commit

* first rebase

* Favorite list (#51)

* Changing column type and updaing input form

cuisine types and atmostphere are string of array that can take multiples input

fix card

WIP front-end design

finish with restaurant type

* update Show UI

* partial rendering for Show page

* get location with geocoder and activejob

* add google map

* partial rendering for Show page

rubucop

redriect after create

* rubocup

* get location with geocoder and add google map

* fix image height

* add validation

* rubocop

* fix image ratio

* fix conflict

* fix credential

* add basic favorite list function

* rubocop

* check user login

* update tailwindcss

* WIP favorite list index page

* toggle favorite button

remove console log

rubocop

* WIP fix reload page

* add section filter

* minor fix

* finish basic section filter

* WIP subtoggle

* WIP index.html for favorite list

* make favorite button an object

* fix redirect to top of the page

* check

* WIP favorite list index page

* index page for favorite list

index page for favorite list

* fix layout

* fix conflict

* add association with hotel and site

* add link for testing function

* fix typo

* add authorization

* fix credentails

* Update database.yml

* delete duplicate program of JS index

* add background judgment

* fix rebase conflicts

* fixed rebase issues

* remove unnecessary photos

* fixed migration timing conflict.

* resolve rebase conflicts

* added role column to user

* added faker data and fixed order show issue

* add excon (~> 0.71)

* delete uploaders img

* modify Uploader missing problem

* resolve merge conflicts

* Remove the "p" for the test

* modify restaurant, site, hotel, plan path

* WIP

* add dashboard_users page

* dashboard_users Done

* modify path

* Remove unnecessary paths and add a background link

* add navbar

* Favorite list (#51)

* Changing column type and updaing input form

cuisine types and atmostphere are string of array that can take multiples input

fix card

WIP front-end design

finish with restaurant type

* update Show UI

* partial rendering for Show page

* get location with geocoder and activejob

* add google map

* partial rendering for Show page

rubucop

redriect after create

* rubocup

* get location with geocoder and add google map

* fix image height

* add validation

* rubocop

* fix image ratio

* fix conflict

* fix credential

* add basic favorite list function

* rubocop

* check user login

* update tailwindcss

* WIP favorite list index page

* toggle favorite button

remove console log

rubocop

* WIP fix reload page

* add section filter

* minor fix

* finish basic section filter

* WIP subtoggle

* WIP index.html for favorite list

* make favorite button an object

* fix redirect to top of the page

* check

* WIP favorite list index page

* index page for favorite list

index page for favorite list

* fix layout

* fix conflict

* add association with hotel and site

* add link for testing function

* fix typo

* add authorization

* fix credentails

* Update database.yml

* fixed migration timing conflict.

* added faker data and fixed order show issue

* add excon (~> 0.71)

* resolve rebase conflicts

---------

Co-authored-by: Tiger Weng <112145785+CCTWeng@users.noreply.github.com>

YuTingLan committed:
Dashboard rebase dev (#55)

* WIP

* add dashboard_users page

* dashboard_users Done

* modify path

* used rubocop

* Remove unnecessary paths and add a background link

* mail send is changed from google to mailgun

* modify mailgun api key

* add navbar

* used rubocop

* Can't switch branches, only commit

* first rebase

* Favorite list (#51)

* Changing column type and updaing input form

cuisine types and atmostphere are string of array that can take multiples input

fix card

WIP front-end design

finish with restaurant type

* update Show UI

* partial rendering for Show page

* get location with geocoder and activejob

* add google map

* partial rendering for Show page

rubucop

redriect after create

* rubocup

* get location with geocoder and add google map

* fix image height

* add validation

* rubocop

* fix image ratio

* fix conflict

* fix credential

* add basic favorite list function

* rubocop

* check user login

* update tailwindcss

* WIP favorite list index page

* toggle favorite button

remove console log

rubocop

* WIP fix reload page

* add section filter

* minor fix

* finish basic section filter

* WIP subtoggle

* WIP index.html for favorite list

* make favorite button an object

* fix redirect to top of the page

* check

* WIP favorite list index page

* index page for favorite list

index page for favorite list

* fix layout

* fix conflict

* add association with hotel and site

* add link for testing function

* fix typo

* add authorization

* fix credentails

* Update database.yml

* delete duplicate program of JS index

* add background judgment

* fix rebase conflicts

* fixed rebase issues

* remove unnecessary photos

* fixed migration timing conflict.

* resolve rebase conflicts

* added role column to user

* added faker data and fixed order show issue

* add excon (~> 0.71)

* delete uploaders img

* modify Uploader missing problem

* resolve merge conflicts

* Remove the "p" for the test

* modify restaurant, site, hotel, plan path

* WIP

* add dashboard_users page

* dashboard_users Done

* modify path

* Remove unnecessary paths and add a background link

* add navbar

* Favorite list (#51)

* Changing column type and updaing input form

cuisine types and atmostphere are string of array that can take multiples input

fix card

WIP front-end design

finish with restaurant type

* update Show UI

* partial rendering for Show page

* get location with geocoder and activejob

* add google map

* partial rendering for Show page

rubucop

redriect after create

* rubocup

* get location with geocoder and add google map

* fix image height

* add validation

* rubocop

* fix image ratio

* fix conflict

* fix credential

* add basic favorite list function

* rubocop

* check user login

* update tailwindcss

* WIP favorite list index page

* toggle favorite button

remove console log

rubocop

* WIP fix reload page

* add section filter

* minor fix

* finish basic section filter

* WIP subtoggle

* WIP index.html for favorite list

* make favorite button an object

* fix redirect to top of the page

* check

* WIP favorite list index page

* index page for favorite list

index page for favorite list

* fix layout

* fix conflict

* add association with hotel and site

* add link for testing function

* fix typo

* add authorization

* fix credentails

* Update database.yml

* fixed migration timing conflict.

* added faker data and fixed order show issue

* add excon (~> 0.71)

* resolve rebase conflicts

---------

Co-authored-by: Tiger Weng <112145785+CCTWeng@users.noreply.github.com>

YuTingLan committed:
Dashboard rebase dev (#55)

* WIP

* add dashboard_users page

* dashboard_users Done

* modify path

* used rubocop

* Remove unnecessary paths and add a background link

* mail send is changed from google to mailgun

* modify mailgun api key

* add navbar

* used rubocop

* Can't switch branches, only commit

* first rebase

* Favorite list (#51)

* Changing column type and updaing input form

cuisine types and atmostphere are string of array that can take multiples input

fix card

WIP front-end design

finish with restaurant type

* update Show UI

* partial rendering for Show page

* get location with geocoder and activejob

* add google map

* partial rendering for Show page

rubucop

redriect after create

* rubocup

* get location with geocoder and add google map

* fix image height

* add validation

* rubocop

* fix image ratio

* fix conflict

* fix credential

* add basic favorite list function

* rubocop

* check user login

* update tailwindcss

* WIP favorite list index page

* toggle favorite button

remove console log

rubocop

* WIP fix reload page

* add section filter

* minor fix

* finish basic section filter

* WIP subtoggle

* WIP index.html for favorite list

* make favorite button an object

* fix redirect to top of the page

* check

* WIP favorite list index page

* index page for favorite list

index page for favorite list

* fix layout

* fix conflict

* add association with hotel and site

* add link for testing function

* fix typo

* add authorization

* fix credentails

* Update database.yml

* delete duplicate program of JS index

* add background judgment

* fix rebase conflicts

* fixed rebase issues

* remove unnecessary photos

* fixed migration timing conflict.

* resolve rebase conflicts

* added role column to user

* added faker data and fixed order show issue

* add excon (~> 0.71)

* delete uploaders img

* modify Uploader missing problem

* resolve merge conflicts

* Remove the "p" for the test

* modify restaurant, site, hotel, plan path

* WIP

* add dashboard_users page

* dashboard_users Done

* modify path

* Remove unnecessary paths and add a background link

* add navbar

* Favorite list (#51)

* Changing column type and updaing input form

cuisine types and atmostphere are string of array that can take multiples input

fix card

WIP front-end design

finish with restaurant type

* update Show UI

* partial rendering for Show page

* get location with geocoder and activejob

* add google map

* partial rendering for Show page

rubucop

redriect after create

* rubocup

* get location with geocoder and add google map

* fix image height

* add validation

* rubocop

* fix image ratio

* fix conflict

* fix credential

* add basic favorite list function

* rubocop

* check user login

* update tailwindcss

* WIP favorite list index page

* toggle favorite button

remove console log

rubocop

* WIP fix reload page

* add section filter

* minor fix

* finish basic section filter

* WIP subtoggle

* WIP index.html for favorite list

* make favorite button an object

* fix redirect to top of the page

* check

* WIP favorite list index page

* index page for favorite list

index page for favorite list

* fix layout

* fix conflict

* add association with hotel and site

* add link for testing function

* fix typo

* add authorization

* fix credentails

* Update database.yml

* fixed migration timing conflict.

* added faker data and fixed order show issue

* add excon (~> 0.71)

* resolve rebase conflicts

---------

Co-authored-by: Tiger Weng <112145785+CCTWeng@users.noreply.github.com>

YuTingLan committed:
Dashboard rebase dev (#55)

* WIP

* add dashboard_users page

* dashboard_users Done

* modify path

* used rubocop

* Remove unnecessary paths and add a background link

* mail send is changed from google to mailgun

* modify mailgun api key

* add navbar

* used rubocop

* Can't switch branches, only commit

* first rebase

* Favorite list (#51)

* Changing column type and updaing input form

cuisine types and atmostphere are string of array that can take multiples input

fix card

WIP front-end design

finish with restaurant type

* update Show UI

* partial rendering for Show page

* get location with geocoder and activejob

* add google map

* partial rendering for Show page

rubucop

redriect after create

* rubocup

* get location with geocoder and add google map

* fix image height

* add validation

* rubocop

* fix image ratio

* fix conflict

* fix credential

* add basic favorite list function

* rubocop

* check user login

* update tailwindcss

* WIP favorite list index page

* toggle favorite button

remove console log

rubocop

* WIP fix reload page

* add section filter

* minor fix

* finish basic section filter

* WIP subtoggle

* WIP index.html for favorite list

* make favorite button an object

* fix redirect to top of the page

* check

* WIP favorite list index page

* index page for favorite list

index page for favorite list

* fix layout

* fix conflict

* add association with hotel and site

* add link for testing function

* fix typo

* add authorization

* fix credentails

* Update database.yml

* delete duplicate program of JS index

* add background judgment

* fix rebase conflicts

* fixed rebase issues

* remove unnecessary photos

* fixed migration timing conflict.

* resolve rebase conflicts

* added role column to user

* added faker data and fixed order show issue

* add excon (~> 0.71)

* delete uploaders img

* modify Uploader missing problem

* resolve merge conflicts

* Remove the "p" for the test

* modify restaurant, site, hotel, plan path

* WIP

* add dashboard_users page

* dashboard_users Done

* modify path

* Remove unnecessary paths and add a background link

* add navbar

* Favorite list (#51)

* Changing column type and updaing input form

cuisine types and atmostphere are string of array that can take multiples input

fix card

WIP front-end design

finish with restaurant type

* update Show UI

* partial rendering for Show page

* get location with geocoder and activejob

* add google map

* partial rendering for Show page

rubucop

redriect after create

* rubocup

* get location with geocoder and add google map

* fix image height

* add validation

* rubocop

* fix image ratio

* fix conflict

* fix credential

* add basic favorite list function

* rubocop

* check user login

* update tailwindcss

* WIP favorite list index page

* toggle favorite button

remove console log

rubocop

* WIP fix reload page

* add section filter

* minor fix

* finish basic section filter

* WIP subtoggle

* WIP index.html for favorite list

* make favorite button an object

* fix redirect to top of the page

* check

* WIP favorite list index page

* index page for favorite list

index page for favorite list

* fix layout

* fix conflict

* add association with hotel and site

* add link for testing function

* fix typo

* add authorization

* fix credentails

* Update database.yml

* fixed migration timing conflict.

* added faker data and fixed order show issue

* add excon (~> 0.71)

* resolve rebase conflicts

---------

Co-authored-by: Tiger Weng <112145785+CCTWeng@users.noreply.github.com>

YuTingLan committed:
Dashboard rebase dev (#55)

* WIP

* add dashboard_users page

* dashboard_users Done

* modify path

* used rubocop

* Remove unnecessary paths and add a background link

* mail send is changed from google to mailgun

* modify mailgun api key

* add navbar

* used rubocop

* Can't switch branches, only commit

* first rebase

* Favorite list (#51)

* Changing column type and updaing input form

cuisine types and atmostphere are string of array that can take multiples input

fix card

WIP front-end design

finish with restaurant type

* update Show UI

* partial rendering for Show page

* get location with geocoder and activejob

* add google map

* partial rendering for Show page

rubucop

redriect after create

* rubocup

* get location with geocoder and add google map

* fix image height

* add validation

* rubocop

* fix image ratio

* fix conflict

* fix credential

* add basic favorite list function

* rubocop

* check user login

* update tailwindcss

* WIP favorite list index page

* toggle favorite button

remove console log

rubocop

* WIP fix reload page

* add section filter

* minor fix

* finish basic section filter

* WIP subtoggle

* WIP index.html for favorite list

* make favorite button an object

* fix redirect to top of the page

* check

* WIP favorite list index page

* index page for favorite list

index page for favorite list

* fix layout

* fix conflict

* add association with hotel and site

* add link for testing function

* fix typo

* add authorization

* fix credentails

* Update database.yml

* delete duplicate program of JS index

* add background judgment

* fix rebase conflicts

* fixed rebase issues

* remove unnecessary photos

* fixed migration timing conflict.

* resolve rebase conflicts

* added role column to user

* added faker data and fixed order show issue

* add excon (~> 0.71)

* delete uploaders img

* modify Uploader missing problem

* resolve merge conflicts

* Remove the "p" for the test

* modify restaurant, site, hotel, plan path

* WIP

* add dashboard_users page

* dashboard_users Done

* modify path

* Remove unnecessary paths and add a background link

* add navbar

* Favorite list (#51)

* Changing column type and updaing input form

cuisine types and atmostphere are string of array that can take multiples input

fix card

WIP front-end design

finish with restaurant type

* update Show UI

* partial rendering for Show page

* get location with geocoder and activejob

* add google map

* partial rendering for Show page

rubucop

redriect after create

* rubocup

* get location with geocoder and add google map

* fix image height

* add validation

* rubocop

* fix image ratio

* fix conflict

* fix credential

* add basic favorite list function

* rubocop

* check user login

* update tailwindcss

* WIP favorite list index page

* toggle favorite button

remove console log

rubocop

* WIP fix reload page

* add section filter

* minor fix

* finish basic section filter

* WIP subtoggle

* WIP index.html for favorite list

* make favorite button an object

* fix redirect to top of the page

* check

* WIP favorite list index page

* index page for favorite list

index page for favorite list

* fix layout

* fix conflict

* add association with hotel and site

* add link for testing function

* fix typo

* add authorization

* fix credentails

* Update database.yml

* fixed migration timing conflict.

* added faker data and fixed order show issue

* add excon (~> 0.71)

* resolve rebase conflicts

---------

Co-authored-by: Tiger Weng <112145785+CCTWeng@users.noreply.github.com>

YuTingLan committed:
fixed member login and resolved conflicts. (#94)

YuTingLan committed:
fixed member login and resolved conflicts. (#94)

YuTingLan committed:
fixed member login and resolved conflicts. (#94)

YuTingLan committed:
fixed pagination display issue

YuTingLan commented:
修正選單搜尋時，若 < 6張卡片，會出現分頁的問題

YuTingLan committed:
revise bad dev

YuTingLan committed:
fixed merge errors

YuTingLan committed:
fixed merge issues

YuTingLan committed:
Merge branch 'dev' into add_page_for_RSH

YuTingLan commented:
這一版只修正這裡
<img width="474" alt="截圖 2023-05-12 下午5 44 26" src="https://github.com/TourMind/tourmind/assets/127638342/b116db2a-a13f-4c58-b2fe-8f7110a740c4">
